backend/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "dependencies": {
+    "body-parser": "^1.14.2",
```

Comment:
Remove all the hats. Only libraries should have hats or tildes in the dependency versioning.